platforms/hyperledger-besu/configuration/samples/network-proxy-none.yaml

```diff
@@ -14,6 +14,9 @@ network:
   # to join an existing network.
   type: besu
   version: 21.10.6  #this is the version of Besu docker image that will be deployed.
+  # onchain permissioning - ref https://besu.hyperledger.org/en/stable/private-networks/tutorials/permissioning/onchain
+  permissioning:
```

Comment:
Please also mention that this field is set to be disabled by default (once you do permissioning: enabled/disabled)

Reply:
updated